pkg/pkgmgmt/client/helpers.go

```diff
@@ -20,20 +21,23 @@ type TestPackageManager struct {
 	RunAssertions []func(pkgContext *portercontext.Context, name string, commandOpts pkgmgmt.CommandOptions) error
 
 	// called keeps track of which mixins/plugins were called
-	called map[string]int
+	called sync.Map
+	lock   sync.Mutex
 }
 
 // GetCalled tracks how many times each package was called
-func (p *TestPackageManager) GetCalled() map[string]int {
-	return p.called
+func (p *TestPackageManager) GetCalled(mixin string) int {
+	calls, _ := p.called.Load(mixin)
```

Comment:
should we check the boolean returned by Load to make sure the mixin exist in the map before we do the type assertion?